app/posts/manage-teacher-training-applications/2021-01-06-closing-an-application.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Closing an application (withdrawal versus rejection)
+description: Attempting to improve the way to close an application at any time
+date: 2021-01-06
+related:
+  items:
+  - text: Reasons for closing an application
+    href: https://docs.google.com/spreadsheets/d/1lKDA7E7PLvp6pC2lUWqSOK4AycfpLV0TOpkp_o-R4nY/edit#gid=0
+    description: Map of what reasons are available at what stage of the application process
+---
+
+## Background
+
+Candidates can end their application by:
+
+- withdrawing their application (application withdrawn)
+- declining an offer (declined)
+
+They might also request their application to be withdrawn which would currently have to be done in Support.
+
+Providers can close an application by:
+
+- rejecting an application (rejected)
+- not responding to an application before the deadline (rejected)
+- withdrawing an offer (offer withdrawn)
+- marking a condition as not met (conditions not met)
+
+Providers give candidates different types of feedback depending on what they’re doing. For example, when rejecting a candidate, providers fill out structured reasons for rejection. Whereas, when withdrawing an offer, they get a free text box.
+
+It’s not always clear to providers what the difference is between rejection and withdrawal.
+
+And an application can not be always be closed depending on the status. For example, if a candidate stops responding after they accept an offer, there’s no way to close the application besides marking a condition as not met.
+
+## Why are we doing this?
+
+We’re doing this work to:
+
+- make it easier for providers to choose the right action without getting hung up on terminology
+- let providers close an application at any time throughout the application process
+- allow providers to process applications where candidates have disappeared or stopped responding
+- give providers confidence that they won’t get punished about rejecting an application
+- give candidates useful feedback about why they were unsuccessful
+- allow candidates to be withdrawn at their request
+- make sure DfE can accurately differentiate between withdrawals and rejections
+
+## What outcomes are we looking for
+
+- less or no support requests to close applications
+- less or no applications being incorrectly marked as automatically rejected or automatically declined
+- more candidates applying again and being successful when they do
+
+## What we did and why
+
+- Replaced the words ‘reject application’ and ‘withdraw offer’ to ‘Close application’ so that users do not get hung up on terminology
+- Replace ‘Declined’, ‘Rejected’, ‘Offer withdrawn’ and ‘Application withdrawn’ statuses with ‘Closed’
+- Always let users close an application (unless it’s already been closed) so that there’s a consistent way to close an application regardless of the status
```

Comment:
```suggestion
- Always let users close an application (unless it’s already been closed) so that there’s a consistent way to close an application regardless of the status.
```